SparkPost/2web2ui
# FE-687/UX-124 Add Calibre as a static asset, Add font-face sheet

### What Changed
- Hosts fonts
- Conversion from gotham > calibre is in another ticket

### How To Test
- This should not change the bundle, but you can verify the fonts work by adding this CSS anywhere
```css
@import url('/static/fonts/calibre/calibre.css');

* {
  font-family: Calibre !important;
}
```
- Can check network/sources tab in chrome dev tools to confirm after that

### To Do
